TACC/Core-CMS
# FP-1648: Update for Core-Styles API Change

## To Do

- [x] Pin `@tacc/core-styles` to commit or version, not [branch](https://github.com/TACC/Core-CMS/pull/489/files#r881827815). 66c6d14

## Overview

Update build command since [TACC/Core-Styles](https://github.com/TACC/Core-Styles) API change.

## Related

- [FP-1648](https://jira.tacc.utexas.edu/browse/FP-1648)
- requires https://github.com/TACC/Core-Styles/pull/23

## Changes

- __change `--input-dir` and `--output-dir` to `--input` and `--output`__﹡
- load branch of https://github.com/TACC/Core-Styles/pull/23
- _remove extra whitespace_

<sub>﹡ This makes it clearer that a user can specify individual files.</sub>

## Testing

```
npm ci
npm run build
```

1. See how many stylesheets are built on `main` branch.
2. See how many stylesheets are built on __this__ branch.
3. Verify same number of files is output as before this PR.

## Screenshots

N/A
